lib/ansible/modules/system/aix_inittab.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Joris Weijters <joris.weijters@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+
+DOCUMENTATION = '''
+---
+author: "Joris Weijters (@molekuul)"
+module: aix_inittab
+short_description: Manages the inittab at AIX.
+description:
+    - Manages the inittab at AIX.
+version_added: "2.3"
+options:
+  name:
+    description: Name of the inittab entry.
+    required: True
+    alias: service
+    type: string
+    default: null
+  runlevel:
+    description: Runlevel of the entry.
+    required: True
+    type: string
+    default: null
+  processaction:
+    description: Action what the init has to do with this entry.
+    required: True
+    choices: [
+               'respawn',
+               'wait',
+               'once',
+               'boot',
+               'bootwait',
+               'powerfail',
+               'powerwait',
+               'off',
+               'hold',
+               'ondemand',
+               'initdefault',
+               'sysinit'
+              ]
+    type: string
+    default: null
+  command:
+    description: What command has to run.
+    requred: True
+    type: string
+    default: null
+  follow:
+    description: After which inittabline shoud the new entry follow.
+    required: False
```

Comment:
required: False, and default: null can be left out. These are the default anyway, and it's easier to read when only the specific stuff is in the overview.